tests-functional/tests/test_positive_values_user_flow.py

```diff
@@ -0,0 +1,82 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
+from pages.all_aboard import AllAboard
+from pages.firstrun import Firstrun
+from pages.sidebar import Sidebar
+
+
+@pytest.fixture
+def complete_firstrun(base_url, selenium):
+    """ Completes /firstrun interactions to start on-boarding"""
+    page = Firstrun(selenium, base_url).open()
+    page.complete_firstrun_values_flow()
+    return page
```

Comment:
It depends on where the fixture is defined. If it's in a conftest.py for example at the test root, then it will apply to all tests. A conftest.py in a subdirectory will apply to tests within that subdirectory. In a test file it will apply to all tests in the file, and in a test class it would apply just to tests in that class.